### ISSUE TITLE
Typing error with 1.0.2

### ISSUE THREAD

**Issue by bard** [redux-utilities/reduce-reducers#30]
Error:

```
node_modules/reduce-reducers/index.d.ts:7:1 - error TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.

7 function reduceReducers<S>(initialState: S | null, ...reducers: Reducer<S>[]): Reducer<S>;
  ~~~~~~~~

node_modules/reduce-reducers/index.d.ts:10:16 - error TS2552: Cannot find name 'reduceReducer'. Did you mean 'reduceReducers'?

10 export default reduceReducer;
                  ~~~~~~~~~~~~~

  node_modules/reduce-reducers/index.d.ts:7:10
    7 function reduceReducers<S>(initialState: S | null, ...reducers: Reducer<S>[]): Reducer<S>;
               ~~~~~~~~~~~~~~
    'reduceReducers' is declared here.


Found 2 errors.
```

index.ts:

```js
import reduceReducers from 'reduce-reducers'
```

tsconfig.json:

```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "commonjs",
    "lib": ["es5", "dom"],
    "jsx": "react",
    "declaration": true,
    "declarationDir": "dist",
    "outDir": "dist",
    "strict": true,
    "allowSyntheticDefaultImports": true
  }
}
```

TypeScript: 3.4.3

**Comment by tuuling** [comment]
The export was left incorrectly as singular “reducer”

**Comment by timche** [comment]
Fixed with [v1.0.3](https://github.com/redux-utilities/reduce-reducers/releases/tag/v1.0.3)
